internal/store/node.go

```diff
@@ -56,9 +57,43 @@ func nodeMetricFamilies(allowAnnotationsList, allowLabelsList []string) []genera
 		createNodeStatusCapacityFamilyGenerator(),
 		createNodeStatusConditionFamilyGenerator(),
 		createNodeStateAddressFamilyGenerator(),
+		createNodeStatusImagesFamilyGenerator(),
 	}
 }
 
+func createNodeStatusImagesFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGeneratorWithStability(
+		"kube_node_status_images",
+		"Container Images on the Node",
+		metric.Gauge,
+		basemetrics.STABLE,
```

Comment:
Should be experimental

Reply:
Just in case I do it wrong: The basemetrics equivalent to experimental is ALPHA? Sorry if this question seems a bit stupid.

Reply:
Thanks. Done.